utils.sh

```diff
@@ -61,6 +61,40 @@ function configure_chronyd() {
   fi
 }
 
+function bootstrap_static_ip_ignition(){
+  ASSESTS_DIR=$1
+if [[ "${IP_STACK}" = "v6" ]]; then
+  BOOTSTRAP_IP=$(nth_ip $EXTERNAL_SUBNET_V6 $((idx + 9)))
+  BOOTSTRAP_PREFIX=$(ipcalc --prefix $EXTERNAL_SUBNET_V6 | cut -d= -f2)
+  BOOTSTRAP_PROTOCOL="ipv6"
+else
+  # Note we assume v4 for dual-stack v4v6 since it's the primary network
+  BOOTSTRAP_IP=$(nth_ip $EXTERNAL_SUBNET_V4 $((idx + 9)))
+  BOOTSTRAP_PREFIX=$(ipcalc --prefix $EXTERNAL_SUBNET_V4 | cut -d= -f2)
+  BOOTSTRAP_PROTOCOL="ipv4"
+fi
+  BOOTSTRAP_CONFIG="[connection]
+type=ethernet
+interface-name=ens3
```

Comment:
change to variable and add env variable for boot ex interface

Reply:
This is the VM NIC (not the provisioning bridge on the provisioning host), so it never changes and can be hard-coded?